spring-cloud-gcp-bigquery/src/test/java/com/google/cloud/spring/bigquery/BigQueryTemplateTest.java

```diff
@@ -223,6 +228,44 @@ void writeJsonStreamTest()
     assertEquals(0, apiRes.getErrors().size());
   }
 
+  @Test
+  void writeJsonStreamNegativeTest()
+      throws DescriptorValidationException, IOException, InterruptedException, ExecutionException {
+    InputStream jsonInputStream = new ByteArrayInputStream(newLineSeperatedJson.getBytes());
```

Comment:
Should this (and the input stream in writeJsonStreamThrowTest) be in a try-with-resources block?

Reply:
It shouldn't cause problems as it's a stream out of a local resource, still, I have updated it with an outer try-with-resources block